### ISSUE TITLE
No support for non-radius accounts

### ISSUE THREAD

**Issue by Kurocon** [Inter-Actief/alexia#53]
Inter-Actief has some members that do not have a RADIUS account (e.g. old-members that are no longer a student, members from the Saxion or other study associations in the country, etc.)

These accounts, while they have a mandate with us to pay for consumptions, cannot be registered into Alexia because it requires a RADIUS account name. It should be possible to let these people also be registered so they can also benefit from our NFC-system.

**Comment by eggied97** [comment]
This problem is gonna be a bit bigger now since RADIUS is going to be EOL from the end of December forward.

**Comment by Kurocon** [comment]
> This problem is gonna be a bit bigger now since RADIUS is going to be EOL from the end of December forward.

Not really, because #73 and #74 implement SAML logins (which is the system that replaces RADIUS) which will be merged soon. Still no support for external accounts (outside of the UT) though for customers, because the s- or m-numbers are used as a synchronization key towards associations. To change that, we would need to agree on a different key to synchronize with, either in addition to the s- and m-numbers, or as a replacement.

**Comment by WKobes** [comment]
While Radius is gone, local accounts are still required for non-UT accounts and development environments that cannot access SAML

**Comment by bramvandartel** [comment]
Wouldn't it be a solution to create an API endpoint in which you should not provide a username where alexia generates one and passes it back? There might be a slight chance that some people will have multiple accounts at multiple associations, but that would not matter that much.

The login functionality would not work for these people, but again, that does not really matter here I think.
